types/pkgcloud/index.d.ts

```diff
@@ -2,8 +2,9 @@
 // Project: https://github.com/pkgcloud/pkgcloud#readme
 // Definitions by: Daniel Friesen <https://github.com/dantman>
 //                 Adam Smith <https://github.com/ScriptSmith>
+//                 Cornelis Heuperman <https://github.com/heuperman>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 2.7
```

Comment:
Why is the version being updated?

Reply:
In TypeScript versions <2.7 string values of the `region` field in `RackspaceProviderOptions` that match those specified in `RackspaceRegions` are not recognised as being of the type `RackspaceRegions` but just as `string` and consequently throw an error.

If I'm not mistaken this is the relevant documentation for the change in version 2.7 that allows this to work :
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#improved-type-inference-for-object-literals